docs/runner/configuration/filtering.md

```diff
@@ -107,6 +107,9 @@ Filters tests by their FQTN which is `$package.$class#$method`. The `#` sign is
 ### "fully-qualified-class-name"
 Filters tests by their FQCN which is `$package.$class`                                     |
 
+### "simple-test-name"
```

Comment:
The name is a bit confusing. I expect to see only the test name. Maybe it is better to call `simple-test-class-and-test-name`?

Reply:
Your logic is good if you know all the options and you know the reasons behind each name.
My message is that the name should be clear for a developer without the context of other names.
Anyway, it is up to you.